liesel/goose/builder.py

```diff
@@ -222,9 +237,53 @@ def build(self) -> Engine:
             if not ker.identifier:
                 ker.identifier = f"kernel_{idx:02d}"
 
+        # set initial values
+        model_states = self._model_state.expect("Model state must be set")
+        # init_strategies = self._init_strategies.value
+        init_strategies = self._init_strategies
+
+        # not_present_strategies = []
+        # for pos_key in pos_keys:
+        #     if pos_key not in init_strategies:
+        #         not_present_strategies.append(pos_key)
+
+        # if not_present_strategies:
+        #     raise Warning(
+        #         f"The strategies for {not_present_strategies} have not been set. "
+        #         "The values will be jittered with a U(-1, 1) noise"
+        #     )
+
+        for pos_key in pos_keys:
+            if pos_key not in init_strategies.keys():
+                init_strategies[
+                    pos_key
+                ] = lambda key, current_value: current_value + jax.random.uniform(
+                    key=key,
+                    shape=current_value.shape,
+                    dtype=current_value.dtype,
+                    minval=-1.0,
+                    maxval=1.0,
+                )
```

Comment:
Don't jitter by default. If `init_strategies.is_none()`, just skip jittering altogether. If  `init_strategies.is_some()`, jitter the parameters that have a strategy, and skip those that don't have one with a warning.

Reply:
Should I raise a warning if they are not set?

Reply:
You mean if `init_strategies.is_none()`? Yes, maybe that's a good idea too...

Reply:
Sure, but we should also handle that in the `DistRegBuilder`

Reply:
Ready for a second check